src/rules/noImplicitDependenciesRule.ts

```diff
@@ -64,9 +76,15 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 
     public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        let whitelist = this.ruleArguments.find((arg) => Array.isArray(arg)) as string[];
```

Comment:
you should be able to do:
`const whitelist = this.ruleArguments.find((arg) => Array.isArray(arg)) || [];`

Reply:
This gives a couple of errors:

```
ERROR: 79:83  strict-boolean-expressions  This type is not allowed in the operand for the '||' operator because it is always truthy. Allowed types are boolean or boolean-or-undefined.
ERROR: 84:13  no-unsafe-any               Unsafe use of expression of type 'any'.
```